src/ovp_pipeline/source_signals/arxiv.py

```
@@ -0,0 +1,128 @@
+        arxiv_id = m.group("id")
+        url = _API.format(id=arxiv_id)
+        req = urllib.request.Request(url, headers={"User-Agent": self.user_agent})
+        try:
+            with urllib.request.urlopen(req, timeout=_TIMEOUT_S) as r:
+                xml_text = r.read()
+        except (urllib.error.URLError, TimeoutError) as exc:
+            logger.warning("arXiv API error for %s: %s", arxiv_id, exc)
+            return None
+
+        try:
+            root = ET.fromstring(xml_text)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)